bridges/snowbridge/pallets/system-frontend/src/lib.rs

```diff
@@ -220,6 +240,57 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		/// Add an additional relayer tip for a committed message identified by `message_id`.
+		/// The tip asset will be swapped for ether.
+		#[pallet::call_index(2)]
+		#[pallet::weight(
+			T::WeightInfo::add_tip()
+				.saturating_add(T::BackendWeightInfo::transact_add_tip())
+		)]
+		pub fn add_tip(origin: OriginFor<T>, message_id: MessageId, asset: Asset) -> DispatchResult
+		where
+			<T as frame_system::Config>::AccountId: Into<Location>,
```

Comment:
Can we even convert straight from AccountId into Location? I would've thought passing in a converter

Reply:
Yep, it is implemented here: https://github.com/paritytech/polkadot-sdk/blob/2567a5ac3a22fe669279277d32561a918a07ae58/polkadot/xcm/src/v5/location.rs#L538